content/posts/newsletter-013/index.md

```diff
@@ -66,6 +66,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [wowAddonManager] v1.0.2
```

Comment:
This will only create a link if you use footnote-style markdown links, e.g:

```markdown
[link text]

[link text]: http://google.com
```

...or:

```markdown
[link text][link-id]

[link-id]: http://google.com
```

See previous issues of the newsletter for examples of this - generally I think @ozkriff prefers for people to use those kinds of links rather than inline ones.

Reply:
I updated the links in commit [1f19c31a5f22eb9c0f410a5ce77b1cffe8969c80](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/247/commits/1f19c31a5f22eb9c0f410a5ce77b1cffe8969c80)